tests/testthat/test-derive_var_atoxgr.R

```diff
@@ -3191,3 +3191,2836 @@ test_that("derive_var_atoxgr, test 81: CTCAEv4 Hypophosphatemia", {
+  "Direct Bilirubin, High",   34.2,   NA,     "umol/L", "3",      21,
+  "Direct Bilirubin, High",   25.66,  NA,     "umol/L", "3",      22,
+  "Direct Bilirubin, High",   25.65,  NA,     "umol/L", "2",      23,
+  "Direct Bilirubin, High",   17.19,  NA,     "umol/L", "2",      24,
+  # ANRHI missing cannot still grade 0 - 1
+  "Direct Bilirubin, High",   17.1,   NA,     "umol/L", NA,       25,
+  "Direct Bilirubin, High",   8,      NA,     "umol/L", NA,       26,
+  "Direct Bilirubin, High",   7.9,    NA,     "umol/L", NA,       27,
+  # AVAL missing cannot grade
+  "Direct Bilirubin, High",   NA,     8,      "umol/L", NA,       28,
+) %>%
+  mutate(
+    BRTHDT = lubridate::ymd("2023-01-01"),
+    LBDT = lubridate::ymd("2023-01-29")
+  )
+
+### add subjects with missing LBDT or BRTHDT
+
+expected_dbilinoage_daids <- expected_dbilile28d_daids %>%
+  filter(TESTNUM %in% c(18, 19)) %>%
+  mutate(
+    LBDT = if_else(TESTNUM == 18, NA, LBDT),
+    BRTHDT = if_else(TESTNUM == 19, NA, BRTHDT),
+    ATOXGRH = NA_character_,
+    TESTNUM = if_else(TESTNUM == 18, 29, 30)
+  )
+
+### put all data together
+expected_dbili_daids <- expected_dbilinoage_daids %>%
+  bind_rows(
+    expected_dbilile28d_daids,
+    expected_dbiligt28d_daids
+  )
+
+
+## Test 90: DAIDS Direct Bilirubin, High ----
+test_that("derive_var_atoxgr Test 90: DAIDS Direct Bilirubin, High", {
+  input_dbili_daids <- expected_dbili_daids %>%
+    select(-ATOXGRH)
+
+  actual_dbili_daids <- derive_var_atoxgr_dir(
+    input_dbili_daids,
+    new_var = ATOXGRH,
+    meta_criteria = atoxgr_criteria_daids,
+    tox_description_var = ATOXDSCH,
+    criteria_direction = "H",
+    get_unit_expr = AVALU
+  )
+
+  expect_dfs_equal(
+    base = expected_dbili_daids,
+    compare = actual_dbili_daids,
+    keys = c("ATOXDSCH", "TESTNUM")
+  )
+})
+
+### Total Bilirubin, High
+
+### > 28 days of age
+
+### Grade 4: >= 5.0 x ULN
+### Grade 3: 2.6 to < 5.0 x ULN
+### Grade 2: 1.6 to < 2.6 x ULN
+### Grade 1: 1.1 to < 1.6 x ULN
+
+expected_tbiligt28d_daids <- tibble::tribble(
+  ~ATOXDSCH,                ~AVAL,  ~ANRHI, ~AVALU,    ~ATOXGRH, ~TESTNUM,
+  "Not a term",             9,      10,     "umol/L",  NA,       1,
+  NA_character_,            9,      10,     "umol/L",  NA,       2,
+  # ANRHI not missing
+  "Total Bilirubin, High",  50,     10,     "umol/L",  "4",      3,
+  "Total Bilirubin, High",  49,     10,     "umol/L",  "3",      4,
+  "Total Bilirubin, High",  26,     10,     "umol/L",  "3",      5,
+  "Total Bilirubin, High",  25.9,   10,     "umol/L",  "2",      6,
+  "Total Bilirubin, High",  16,     10,     "umol/L",  "2",      7,
+  "Total Bilirubin, High",  15.9,   10,     "umol/L",  "1",      8,
+  "Total Bilirubin, High",  11,     10,     "umol/L",  "1",      9,
+  "Total Bilirubin, High",  10.9,   10,     "umol/L",  "0",      10,
+  # Unit missing can grade
+  "Total Bilirubin, High",  10.9,   10,     NA,        "0",      11,
```

Comment:
```suggestion
  "Total Bilirubin, High",  10.9,   10,     NA,        NA,      11,
```

Reply:
As above: "Unit not important as all grades are comparing AVAL against ANRLO, not comparing against a constant value where unit would be important. I will update comment to explain."